helpdesk/email.py

```diff
@@ -84,16 +90,24 @@ def process_email(quiet=False):
             logger.addHandler(log_file_handler)
         else:
             log_file_handler = None
-
-        try:
             if not q.email_box_last_check:
                 q.email_box_last_check = timezone.now() - timedelta(minutes=30)
-
+        try:
             queue_time_delta = timedelta(minutes=q.email_box_interval or 0)
             if (q.email_box_last_check + queue_time_delta) < timezone.now():
                 process_queue(q, logger=logger)
                 q.email_box_last_check = timezone.now()
                 q.save()
+                log_msg: str = f"Queue successfully processed: {q.slug}"
+                logger.info(log_msg)
+                if debug_to_stdout:
+                    print(log_msg)
```

Comment:
Doesn't this mean that if the django logger is configured to log to console then the message will be output twice?

Reply:
It would but this is meant for production debugging and in production iot makes no sense at all to have console logging enabled unless you are redirecting console logs to some other persistant log store.
This mechanism is intended for quick and easy debugging in a production setup and is intended not to depend on any production configuration at all.